proj/src/main/webapp/script.js

```diff
@@ -69,4 +72,35 @@ function pushChatMessage() {
     const message = document.getElementById('message-input').value;
     // push message to datastore
     messageRef.push(message);
-} 
+}
+
+var messages;
```

Comment:
Why is this. defined outside of all other functions? I don't see where we add to this or set this.

Reply:
this was a debug statement i forgot to remove